web/src/main/java/se/inera/intyg/webcert/web/integration/interactions/sendmessagetocare/SendMessageToCareResponderImpl.java

```diff
@@ -51,19 +51,26 @@ public SendMessageToCareResponseType sendMessageToCare(String logicalAddress, Se
             arendeService.processIncomingMessage(ArendeConverter.convert(request));
             result.setResultCode(ResultCodeType.OK);
         } catch (WebCertServiceException e) {
-            result.setResultCode(ResultCodeType.ERROR);
             switch (e.getErrorCode()) {
+                case MESSAGE_ALREADY_EXISTS:
+                    result.setResultCode(ResultCodeType.INFO);
```

Comment:
Should I set setErrorId?

Reply:
This is the response that will be sent to FK. Is this ok?

Reply:
Yes!